components/ui/Rating.tsx

```diff
@@ -23,24 +23,30 @@ import { translate, translateWithParameters } from '../../helpers/l10n';
 import { formatMeasure } from '../../helpers/measures';
 import './Rating.css';
 
-interface Props {
+interface Props extends Pick<React.AriaAttributes, 'aria-label'> {
```

Comment:
Why only 'aria-label' and not other like for the icon in the previous PR?
(I am a noob in aria thing, not sure if other properties could be useful).

Reply:
I don't know, I just thought we only need aria-label so let's only keep this one, but like you I'm a noob at this...
@wouter-admiraal-sonarsource  ? :smile: Do you think it make sens to allow any aria attribute ?

Reply:
Well, ofc, not all ARIA attrs will make sense 🤷‍♂️ . But I'm not sure it's worth it to whitelist them. It's the same with any HTML: you can add any ARIA attribute to any element, but some of them won't make any sense, and will simply be ignored.

I'm personally OK with allowing any ARIA attribute, and let the dev make a call on what makes sense in what condition (e.g., I could imagine `aria-hidden` and `aria-labelledby` also make sense, depending on context).

Reply:
I'll allow all of them then